docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1532,6 +1534,8 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/audit/serviceProperties/geoCounty`|US|
 |`fhirServer/bulkdata/isExportPublic`|true|
 |`fhirServer/bulkdata/validBaseUrlsDisabled`|false|
+|`fhirServer/bulkdata/cosFileMaxResources`|500000|
+|`fhirServer/bulkdata/cosFileMaxSize`|209715200|
```

Comment:
What does this equate to? 2G?

Reply:
200M

Reply:
I think it should be added to the documentation (not necessarily in this row), that it equates to 200M

Reply:
let me add to the comment for this config. after -1.

Reply:
done